### PR TITLE
#86 add adwaita statuspage

### LINKED ISSUE #86: Add `adw.StatusPage` widget
https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/class.StatusPage.html

### PR DESCRIPTION
**Pull request by PhilippMDoerner**
Closes #86 

Adds the adwaita Status Page widget

### CODE REVIEW

File: docs/recommended_tools.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ For an overview over which icons are available, you can install the [Icon Librar
 
 For using custom icons and icon-sets, you may want to look into [Symbolic Preview](https://flathub.org/apps/details/org.gnome.design.SymbolicPreview).
 
+Freedesktop.org also provides a [specification for standard icon-names](https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html#names) that Gnome uses.
```
Comment (PhilippMDoerner) [inline docs/recommended_tools.md:20]: On the eternal search for "Does GTK/Gnome have any kind of official list of icons you can use?" (which I still haven't found a definitive list for), I stumbled over this link and thought it'd be useful enough to include.

Comment (can-lehmann) [reply docs/recommended_tools.md:20]: I usually use the `IconLibrary` application mentioned above, but I would be happy to also include this link.

File: examples/README.md [hunk 2]
```diff
@@ -192,6 +192,10 @@ The `widgets` directory contains examples for how to use different widgets.
     <td><a href="https://github.com/can-lehmann/owlkettle/blob/main/examples/widgets/adw/preferences_group.nim">Preferences Group</a></td>
     <td><img alt="Preferences Group" src="../docs/assets/examples/preferences_group.png" width="721px"></td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/can-lehmann/owlkettle/blob/main/examples/widgets/adw/status_page.nim">Status Page</a></td>
+    <td><img alt="Preferences Group" src="../docs/assets/examples/status_page.png" width="922px"></td>
```
Comment (can-lehmann) [inline examples/README.md:197]: `alt` needs to be updated

File: examples/widgets/adw/status_page.nim [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import owlkettle, owlkettle/[playground, adw]
+
+viewable App:
+  description: string = "Some Description"
+  iconName: string = "weather-clear-symbolic"
+  title: string = "Some Title"
```
Comment (can-lehmann) [inline examples/widgets/adw/status_page.nim:28]:
```suggestion
  title: string = "Some Title"

```

**Comment by can-lehmann** [comment]
Looks good!

**Pull request merged**